Assets/Scripts/SS3D/Networking/Settings/NetworkSettings.cs

```diff
@@ -40,6 +40,12 @@ public sealed class NetworkSettings : ScriptableSettings
 		/// </summary>
 		public ushort ServerPort = 2222;
 
+		/// <summary>
+		/// Enables Fish-net's bandwidth UI in the game.
+		/// </summary>
+		[Header("Debug Settings")]
```

Comment:
that is not related to the PR, remove please

Reply:
Its a very small addition, I see no harm adding it here

Reply:
fine, I prefer having different features in different PR, but that one is very small so, fine

Reply:
k